tests/rootfs_boot.py

```diff
@@ -183,11 +184,18 @@ def boot(self, reflash=True):
 
             ips = []
             for not_used in range(5):
-                ips = [board.get_interface_ipaddr(board.wan_iface)]
+                ip = board.get_interface_ipaddr(board.wan_iface)
+                assert ipaddress.IPv4Address(ip.decode('utf-8')) in prov.cm_network
```

Comment:
Linked to the error reporting improvements, could you add an err message string to the asserts?

Reply:
I'm not seeing this link...

Reply:
linked i.e. with reference to improving error reporting

Reply:
yep, assert, "this has happened maybe do this to fix it"